docs/development/_index.md

```diff
@@ -66,11 +66,18 @@ make app-linux
 ```
 
 ```bash
-make app-windows
+make app-mac
 ```
 
 ```bash
-make app-mac
+make app-win
+```
+
+For Windows, by default it will produce an installer using [NSIS (Nullsoft Scriptable Install System)](https://sourceforge.net/projects/nsis/).
+If you prefer an `.msi` installer, then be sure to install the [WiX toolset](https://wixtoolset.org/), have its `light.exe` and `candle.exe` in the Windows path, and run:
```

Comment:
I have wixtoolset installed with chocolatey, but there is no light.exe or candle.exe. Do you know where it should be?

Reply:
I guess the chocolatey install was broken. The installer worked.

Reply:
I installed with WiX but had to add those executables to Windows's Path.